InformationScripting/src/queries/RuntimeQuery.h

```diff
@@ -41,7 +43,9 @@ class INFORMATIONSCRIPTING_API RuntimeQuery : public LinearQuery
 	private:
 		friend class QueryRegistry;
 
-		RuntimeQuery(Model::Node* target, QStringList);
+		QueryExecutor* executor_{};
+
+		RuntimeQuery(Model::Node* target, QStringList, QueryExecutor* executor = nullptr);
```

Comment:
Should the executor parameter have a default nullptr value? Isn't it better if we don't allow that?